ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -1662,7 +1662,7 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = PriceWidgetExtension.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
You're setting a development profile to a release target.  I think this will break on CI => Testlight => Prod